src/components/sponsor-tiers/sponsor-tiers.tsx

```diff
@@ -191,21 +202,20 @@ export const SponsorTiers = () => {
         ]}
       />
 
-      <div className="space-y-10">
-        <SponsorTier
-          title="Bronze"
-          price={2000}
-          totalSlots={"limited"}
-          features={["Logo & recruiting ad on EuroPython website and more!"]}
-        />
+      <SponsorTier
+        title="Bronze"
+        price={2000}
+        totalSlots={"limited"}
```

Comment:
```suggestion
        totalSlots={"unlimited"}
```

Reply:
Are these unlimited because of the group tickets option?